packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -582,6 +582,7 @@ - (FLTPositionMessage*)position:(FLTTextureMessage*)input error:(FlutterError**)
 - (void)seekTo:(FLTPositionMessage*)input error:(FlutterError**)error {
   FLTVideoPlayer* player = _players[input.textureId];
   [player seekTo:[input.position intValue]];
+  [_registry textureFrameAvailable:input.textureId.intValue];
```

Comment:
I've used this trick, and it doesn't always work. Unfortunately, what can happen is when Flutter comes to request the frame, the iOS AVPlayer can say there's no frame available and this results in a black frame being rendered.

To fix it, I've actually had to unpause the display link and pause it again after 0.5 seconds. It won't pause again if a play was requested in the mean time, and I also handle getting a new seek request in the middle as well.

My solution works perfectly and never results in any black frames.

Reply:
@dannyvalentesonos Sorry for the incredibly late reply, been busy with work.  Can you create a small repo that shows your fix in action?

Reply:
@dannyvalentesonos do you have a sample or reference to your approach?

Reply:
I don't have a repo that only has this change unfortunately. I will try and find some time to get one together.

Reply:
I'm going resolve this.  I've been using this fix for over a year now and the client hasn't had any complaints